authors/apps/authentication/serializers.py

```diff
@@ -13,18 +11,20 @@ class RegistrationSerializer(serializers.ModelSerializer):
     # special characters and
     # no longer than 128
     # characters, and can not be read by the client.
-    password = serializers.RegexField(
-        regex='^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[@#$%^&+=*!])',
-        max_length=128,
-        min_length=8,
-        write_only=True,
-        error_messages={
-            'required': 'Password field required',
-            'min_length': 'Ensure Password field has at least 8 characters',
-            'invalid': 'Password should contain a lowercase, uppercase numeric'
-            ' and special character'
-        }
-    )
+    def password_validate():
```

Comment:
This function can exist outside the class. I think that's a better implementation

Reply:
Let me refactor